### ISSUE TITLE
A11y: Pages on example site should have a level one heading

### ISSUE THREAD

**Issue by arcepaul** [NCIOCPL/ncids#730]
One of the issues flagged by axeDevTools is that pages are missing `h1`s on some pages of the example site.  Find pages without h1 tags and add them so that the tool does not flag them.

**ESTIMATE:** 1
